apps/community/src/components/auth-input.tsx

```diff
@@ -0,0 +1,19 @@
+import * as styles from './auth-input.css';
+
+interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  type: string;
+  placeholder: string;
+}
+
+function AuthInput({ type, placeholder, ...props }: InputProps) {
+  return (
+    <input
+      className={styles.input}
+      type={type}
+      placeholder={placeholder}
+      {...props}
+    />
+  );
+}
```

Comment:
```suggestion
function AuthInput({ type, ...props }: InputProps) {
  return (
    <input
      className={styles.input}
      type={type}
      {...props}
    />
  );
}
```

placeholder를 rest에 넣지 않고 따로 빼서 할당한 이유가 있나요?

Reply:
placeholder가 필수요소라고 생각하고 구현했는데, 생각해보니 필요없는 부분이었어요.
DS에 input 컴포넌트 추가하면서 제거했습니다